### ISSUE TITLE
Create Json File instead of Html

### ISSUE THREAD

**Issue by timpulver** [dpnem/NetlabelList#2]
If all data was in JSON-format instead of HTML, it would be easier to maintain and other people could build stuff upon it without parsing.

**Comment by dpnem** [comment]
Good idea. The only issue I have is people who use full copyright licenses and charge mon y, think that they have a netlabel, when it's just a commercial digital label.

Let me think some more about it. If you have any suggestions, I would definitely like to hear them.

Dave

**Comment by timpulver** [comment]
I’m building a parser for the list right now. Would open up lots of possibilities if the list was well structured!

I know what you mean with non-CC »netlabels« – not sure about this either. I think putting out mostly free releases is enough to call something netlabel. There are many new labels around doing exactly this. They don’t know much about netlabel culture and just call themselves «free label». I think leaving them out would be a mistake.

But I don’t get why you bring up the CC vs. non-CC thing here. Does not make any difference regarding file format (JSON vs. HTML).

**Comment by timpulver** [comment]
I parsed the HTML and created a JSON-file.
[Fork](https://github.com/timpulver/NetlabelList).
Should I send you a pull request?

**Comment by timpulver** [comment]
Seems like your site depends on the HTML in this repository. There are lots of benefits in storing this information in Json instead of Html. I think the proper way would be to generate your Html-site from the Json-file.
